csrc/python_frontend/python_bindings.cpp

```diff
@@ -686,6 +686,48 @@ void defineHeuristicParamBindings(py::module& nvfuser) {
       .PARAM(PointwiseParams, unroll_factor_outer)
       .PARAM(PointwiseParams, lparams);
 
+  // Reduction scheduler parameters
+  INITHEURISTICPARAMS(ReductionParams)
+      .PARAM(ReductionParams, fastest_dim)
```

Comment:
Would it improve readabiity to sort these lines?

Reply:
I added them in order they appear in `ReductionParams`, but I thought alphabetical order is easier to parser.